documentation/modules/post/windows/manage/persistence_service.rb

```diff
@@ -0,0 +1,100 @@
+## Overview
+This Module will generate and upload an executable to a remote host and make it persistent service.
+It will create a new service which will start the payload whenever the service is running. Privilege is required.
+
+## Module Options
```

Comment:
This doesn't follow the typical format we've done in the past.
Also, for many of the 'default' options (not specific or unique to this module), you can remove them. I would remove lhost, lport, payload, session

Reply:
Sorry for not checking the doc format, just copy from some old module.